tests/backstop/scenarios.js

```diff
@@ -1,6 +1,104 @@
 const storybookHost = require( './detect-storybook-host' );
 const rootURL = `${ storybookHost }iframe.html?id=`;
 const storybookStories = require( '../../.storybook/storybook-data' );
+// does not work as these paths are relative to this file
+// const filePaths = require( '../../.storybook/main' );
+const glob = require( 'glob' );
+const fs = require( 'fs' );
+const parser = require( '@babel/parser' );
+const traverse = require( '@babel/traverse' ).default;
+const csf = require( '@componentdriven/csf' );
+
+const camelCaseToKebabCase = ( string ) => string
+	.replace( /([a-z0-9])([A-Z])/g, '$1-$2' )
+	.replace( /([A-Z])([A-Z])(?=[a-z])/g, '$1-$2' )
+	.toLowerCase();
```

Comment:
Needed in order

I see we have some other similar utils in some other places so happy to move this (unless I'm missing something, but as per Asvin's snippet no stories were being found.  The Id needs to be the Url)

Reply:
How about we use [`lodash/kebabCase`](https://lodash.com/docs/4.17.15#kebabCase) instead?

Reply:
Nice. I didn't know that existed